docs/ref/csrf.txt

```diff
@@ -1,10 +1,18 @@
+.. _csrf-protection-reference:
```

Comment:
Use `:doc:` links instead of refs for linking to a document, that makes these headers unnecessary.

Reply:
I've found that using reference targets makes it easier if files will ever be moved or renamed in the future, but can change to a `:doc:` link if preferred.